packages/debug-test/PlaygroundAutomation.ts

```diff
@@ -42,29 +44,189 @@ async function setCheckedState(
   testLog.message(`"${checkBoxName}" has checkbox state ${checkedState}`);
 }
 
+export async function primePackageComboBox() {
+  // On ADO agents, initial expansion of the drop list can result in a UIA tree that's missing nodes for
+  // some list items. We're trying to "prime" the drop list by dropping and collapsing it prior to test
+  // execution.
+  testLog.message('start primePackageComboBox');
+  const entryPointComboBox = (
+    await $$('//Window/ComboBox[@AutomationId="x_entryPointCombo"]')
+  )[0];
+
+  // found to be necessary in ADO runs - perhaps it puts the app into the foreground
+  await entryPointComboBox.click();
+  await sleep(500);
+
+  await entryPointComboBox.click(); // focus combo box
+  await sleep(500);
+
+  await entryPointComboBox.keys(['ArrowDown']); // drop drop list
+  await sleep(500);
+  await entryPointComboBox.keys(['End']); // move cursor to end of edit box
+  await sleep(500);
+  await entryPointComboBox.keys(['End']); //select last drop list item
+  await sleep(500);
+
+  await app.waitUntil(
+    async () =>
+      (
+        await $$(
+          '//Window/ComboBox[@AutomationId="x_entryPointCombo"]/ListItem',
+        )
+      ).length > 1,
+    {timeout: 20000},
+  );
+
+  const listItems = await $$(
+    '//Window/ComboBox[@AutomationId="x_entryPointCombo"]/ListItem',
+  );
+
+  for (let i = 0; i < listItems.length; ++i) {
+    const listItemText = await listItems[i].getText();
+    testLog.message(`text of list item ${i}: "${listItemText}"`);
+  }
```

Comment:
for...of instead ?

Reply:
Written as is allows me to use the loop variable (i) in the log output.